pkg/controllers/vdb/eula.go

```diff
@@ -61,14 +61,8 @@ func acceptEulaInPod(ctx context.Context, pf *PodFact, pRunner cmds.PodRunner) e
 	}
 	tmp.Close()
 
-	_, _, err = pRunner.CopyToPod(ctx, pf.name, names.ServerContainer, tmp.Name(), paths.EulaAcceptanceScript)
-	if err != nil {
-		return err
-	}
-
-	_, _, err = pRunner.ExecInPod(ctx, pf.name, names.ServerContainer, "/opt/vertica/oss/python3/bin/python3", paths.EulaAcceptanceScript)
-	if err != nil {
-		return err
-	}
-	return nil
+	// Copy and execute the script
+	_, _, err = pRunner.CopyToPod(ctx, pf.name, names.ServerContainer, tmp.Name(), paths.EulaAcceptanceScript,
```

Comment:
So now you copy and execute the script in a single function. Is that as you mentioned to reduce the number of "exec"?

Reply:
Yes. Before we would do a 'kubectl exec' to copy the file in. Then another 'kubectl exec' to run the command we copied in. This just combines the two.